flutter_google_places_sdk_platform_interface/lib/src/types/bounds.dart

```diff
@@ -0,0 +1,34 @@
+import 'location.dart';
+
+class Bounds {
+  const Bounds({
+    this.northeast,
+    this.southwest,
+  });
+
+  final Location northeast;
+  final Location southwest;
+
+  @override
+  bool operator ==(Object other) {
+    return identical(this, other) || hashCode == other.hashCode;
+  }
+
+  @override
+  int get hashCode => northeast.hashCode + southwest.hashCode;
+
+  @override
+  String toString() {
+    return 'Bounds{northeast: $northeast, southwest: $southwest}';
+  }
+
+  Map<String, dynamic> toMap() => {
+        'northeast': northeast.toMap(),
+        'southwest': southwest.toMap(),
+      };
+
+  static Bounds fromMap(Map<dynamic, dynamic> map) => Bounds(
+        northeast: Location.fromMap(map['northeast']),
+        southwest: Location.fromMap(map['southwest']),
+      );
+}
```

Comment:
We have the `Viewport` class. Can't we use that one?

Reply:
Oops. Of course.